packages/database/src/env.ts

```diff
@@ -0,0 +1,17 @@
+import z from 'zod'
+
+import 'dotenv/config'
+
+const envvars = z.object({
+  NODE_ENV: z.string(),
+  DATABASE_URL: z.string(),
+  TEST_DATABASE_URL: z.string(),
+})
+
+export default envvars.parse(process.env)
```

Comment:
errors are better handle with the other package, i don't get why you don't use it

Reply:
because I strongly agree with the idea that adding dependencies should be the last option to consider and t3-env brings very [minimal advantages](https://env.t3.gg/docs/introduction#transforms-and-default-values) vs this:
- **defaults and transforms**
  - you should never transform env vars anyways and defaults are nice but hardly a must-have imo
 - **support for multiple environments**
   - can be done with a couple lines of code as done in /api package

Reply:
I think we're reimplementing a small package that does it super nice